0_portfolio_input_check_functions.R

```diff
@@ -418,7 +418,7 @@ check_fin_mapped_sectors <- function(fin_data) {
     ))
 
   actual_sectors <- unique(fin_data$security_mapped_sector)
-  if (any(!actual_sectors %in% c(sector_list, other_sector_list, "Other"))) {
+  if (any(!actual_sectors %in% c(sector_list, pacta_sectors_not_analysed, "Other"))) {
```

Comment:
> [This] could ... affect the web tool scripts via `get_and_clean_fin_data()` -- CJ

Clare can you please explain a bit what is the purpose of this change and if/where these objects exist?

Reply:
looking into this myself a bit... `pacta_sectors_not_analysed` appears to be something that gets set in the parameters yml file, and read in by `set_project_parameters()` in `0_global_functions.R`

https://github.com/2DegreesInvesting/PACTA_analysis/blob/af15e6c2b1d83cc1724d831139b7a7fec3101f8b/0_global_functions.R#L67

and it appears in the default parameter files for each of the projects, so I give a 👍🏻 
https://github.com/2DegreesInvesting/PACTA_analysis/search?q=pacta_sectors_not_analysed

Reply:
additionally, `pacta_sectors_not_analysed` seems to have been introduced here (for context)
https://github.com/2DegreesInvesting/PACTA_analysis/pull/350